packages/ui/v2/core/Shell.tsx

```diff
@@ -162,6 +172,7 @@ type LayoutProps = {
   flexChildrenContainer?: boolean;
   isPublic?: boolean;
   withoutMain?: boolean;
+  settingsSection?: boolean;
```

Comment:
Added this settingsSection prop to the shell instead of using router to check if we were in the settings sections

Reply:
If this will only trigger a display we can use isSettingsSection? as our prop is a boolean.
```suggestion
  isSettingsSection?: boolean;
```